docs/installation.md

```diff
@@ -24,8 +24,9 @@ distribution the required packages are most likely named something similar.
 If the desktop notifications bindings are also installed
 (`gir1.2-notify-0.7` for Debian/Ubuntu),
 you will also get desktop notifications when devices come online/go offline.
-For gnome-shell/Unity support, you also need to have `gir1.2-appindicator3-0.1`
-installed in Debian/Ubuntu.
+For GNOME Shell/Budgie Desktop/KDE/XFCE support, you also need to have
+`gir1.2-ayatanaappindicator3-0.1` installed in Debian/Ubuntu. For Unity support
+in Ubuntu, you need to have `gir1.2-appindicator3-0.1` installed instead.
```

Comment:
Wouldn't it be better to recommend the ayatana version whenever it is available, even for Unity?

Reply:
At first I thought so too. But then I didn't want to forget Unity users. Although Unity is dead and Ubuntu 18.04LTS and newer switched back to GNOME with a Unity look & feel for their default flavour, there is also Ubuntu 16.04LTS  with the real Unity out there which still receives updates for another year. In addition, I don't know how long Canonical will stick to its own more or less unmaintained version of libindicator + libappindicator as these are still the ones that are preinstalled if you choose to have GNOME with unity-session. That was mainly the reason for the second sentence.
So what do you think about the following?
> Although it is always preferable to install and rely on `gir1.2-ayatanaappindicator3-0.1`, `gir1.2-appindicator3-0.1` is also still supported for Unity in Ubuntu.

Anyway, I will change the sentence to whatever you like. So it's  up to you. Let me know your decision.

Reply:
How about
```
Although it is preferable to install and use gir1.2-ayatanaappindicator3-0.1 if it is available, gir1.2-appindicator3-0.1 can be used in its place (e.g., for Unity in Ubuntu).
```

Reply:
That's also fine. Yet I think it's better to exchange "preferable" by "recommended".
However, do you want me to amend my last commit and then force-push the changes or just to add another commit?

Reply:
"recommended" is fine.   

Please amend the comment and force-push.